lib/localparticipant.js

```diff
@@ -187,7 +189,7 @@ class LocalParticipant extends Participant {
     this.on('trackStopped', localTrackStopped);
 
     this._tracks.forEach(track => {
-      this._addLocalTrack(track);
+      this._addLocalTrack(track, 'medium');
```

Comment:
we should move this string into `lib/util/constants.js`, and reference constant here.